core/domain/rights_manager.py

```diff
@@ -124,8 +124,8 @@ def validate(self):
                 has owners, editors, voice artists or viewers specified.
         """
         if self.community_owned:
-            if (self.owner_ids or self.editor_ids or self.voice_artist_ids or
-                    self.viewer_ids):
+            if (self.owner_ids or self.editor_ids or self.voice_artist_ids or (
```

Comment:
Is this expected -- to add an angular bracket at the last or?

Reply:
yes, we can use the bracket here.